lib/membrane_ffmpeg_swscale/scaler/scaler.ex

```diff
@@ -94,11 +98,6 @@ defmodule Membrane.FFmpeg.SWScale.Scaler do
 
   @impl true
   def handle_end_of_stream(:input, _context, state) do
-    {{:ok, end_of_stream: :output, notify: {:end_of_stream, :input}}, state}
-  end
-
-  @impl true
-  def handle_prepared_to_stopped(_context, state) do
-    {:ok, %{state | native_state: nil}}
+    {[end_of_stream: :output], state}
```

Comment:
This is now equivalent to the default implementation, you can remove this function

Reply:
bump